### ISSUE TITLE
Limb crushing being based on damage instead of specific parameters.

### ISSUE THREAD

**Issue by Drogo-man** [studio-minus/ppg-bugs-and-requests#2229]
**Feature description**
Pretty simple, when a limb is crushed it should be based on how much damage it takes in one frame instead of only occurring when hit with multiple bullets at once or when under weight.

**Feature purpose**
The current system is too specific and outdated, so this new system would make it more universal and "realistic".

**Additional context**

**Comment by DindinYT37** [comment]
How else would it take damage that should cause crushing other than force applied by projectiles and/or mass?

**Comment by Drogo-man** [comment]
> How else would damage that should cause crushing other than force applied by projectiles and/or mass happen? I could see pressure caused by being underwater or explosions but that's about it.

It you get hit by a very big bullet, or get blown up at point blank, among other things.

**Comment by DindinYT37** [comment]
> > How else would damage that should cause crushing other than force applied by projectiles and/or mass happen? I could see pressure caused by being underwater or explosions but that's about it.
> 
> It you get hit by a very big bullet, or get blown up at point blank, among other things.

The hell is "very big bullet", wouldn't fall under both projectiles **and** mass? Also, the blown up falls under the pressure I was talking about.

**Comment by Drogo-man** [comment]
Like the 120mm Canon, 30mm heat autocannon, maybe the 50 bmg amr.

**Comment by DindinYT37** [comment]
> Like the 120mm Canon, 30mm heat autocannon, maybe the 50 bmg amr.

Pretty sure limb crushing already happens with those though?

**Comment by Drogo-man** [comment]
No. The guy just gets flung, and his limbs fall off with the 30mm.

**Comment by Drogo-man** [comment]
You're really trying hard to make actually good suggestions not get into the game huh.

**Comment by DindinYT37** [comment]
> No. The guy just gets flung, and his limbs fall off with the 30mm.

If that's so, I guess.


> You're really trying hard to make actually good suggestions not get into the game huh.

I'm trying to make sure the suggestions make actually sense, and you've proven that this does.

**Comment by Drogo-man** [comment]
Ok i guess that's a good thing, but you should tone it down a little since EVERYONE finds you hella annoying.

**Comment by DindinYT37** [comment]
> Ok i guess that's a good thing, but you should tone it down a little since EVERYONE finds you hella annoying.

Quite exaggerated saying everyone, also it's not really my fault most people don't even know how to properly explain their request.
